x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/card_decorators/event_filters_process_descendant_indicator.test.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { EventFiltersProcessDescendantIndicator } from './event_filters_process_descendant_indicator';
+import type { AnyArtifact } from '../../types';
+import type { AppContextTestRender } from '../../../../../common/mock/endpoint';
+import { createAppRootMockRenderer } from '../../../../../common/mock/endpoint';
+import {
+  FILTER_PROCESS_DESCENDANTS_TAG,
+  GLOBAL_ARTIFACT_TAG,
+} from '../../../../../../common/endpoint/service/artifacts/constants';
+import type { ArtifactEntryCardDecoratorProps } from '../../artifact_entry_card';
+
+describe('EventFiltersProcessDescendantIndicator', () => {
+  let appTestContext: AppContextTestRender;
+  let renderResult: ReturnType<AppContextTestRender['render']>;
+  let render: (
+    props: ArtifactEntryCardDecoratorProps
+  ) => ReturnType<AppContextTestRender['render']>;
+
+  const STANDARD_EVENT_FILTER: AnyArtifact = {
+    tags: [GLOBAL_ARTIFACT_TAG],
+  } as Partial<AnyArtifact> as AnyArtifact;
+
+  const PROCESS_DESCENDANT_EVENT_FILTER: AnyArtifact = {
+    tags: [GLOBAL_ARTIFACT_TAG, FILTER_PROCESS_DESCENDANTS_TAG],
+  } as Partial<AnyArtifact> as AnyArtifact;
```

Comment:
can I suggest that you set these explicitly for every test inside of `beforeEach()` -- or -- convert them to functions that return a new value every time.

Reply:
9de6cb1e53bca0d5e0bde67a09195091bf57fab0

Reply:
9de6cb1e53bca0d5e0bde67a09195091bf57fab0